semcore/d3-chart/src/style/dot.shadow.css

```diff
@@ -19,10 +18,6 @@ SDot[hide] {
   display: none;
 }
 
-SDot[active] {
-  r: calc(var(--radius) * 8/6);
```

Comment:
why do you removed this? How will the size of the dots change on hover?

Reply:
Cause I've added `const radius = radiusBase * (active ? 8 / 6 : 1);`